File: 30 Days of Code/03 - Day 2 - Operators.py
# ========================
#       Information
# ========================

# Direct Link: https://www.hackerrank.com/challenges/30-operators/problem
# Difficulty: Easy
# Max Score: 30
# Language: Python

# ========================
#         Solution
# ========================

def solve(meal_cost, tip_percent, tax_percent):
    '''Function to calculate the total cost'''
    tax = meal_cost* tip_percent/100 + meal_cost* tax_percent/100
    total = meal_cost + tax
    print(round(total))

File: 30 Days of Code/C#/03 - Day 2 - Operators.cs
// ========================
//       Information
// ========================

// Direct Link: https://www.hackerrank.com/challenges/30-operators/problem
// Difficulty: Easy
// Max Score: 30
// Language: C#

// ========================
//         Solution
// ========================

using System;

class Solution {
    static void Main(String[] args) {
        var mealCost = double.Parse(Console.ReadLine());
        var tipPercent = int.Parse(Console.ReadLine());
        var taxPercent = int.Parse(Console.ReadLine());

        var tip = tipPercent * mealCost / 100;
        var tax = taxPercent * mealCost / 100;

        var totalCost = Math.Round(tip + tax + mealCost);
        Console.WriteLine(totalCost);
    }
}
